packages/vx-pattern/src/constants/index.ts

```diff
@@ -0,0 +1,7 @@
+export type PatternOrientationType = 'horizontal' | 'vertical' | 'diagonal';
+
+export const PatternOrientation: { [key in PatternOrientationType]: PatternOrientationType } = {
```

Comment:
change
`{ [key in PatternOrientationType]: PatternOrientationType }`
to
`{ [key in PatternOrientationType]: key }`

to enforce that `key === value`

Reply:
nice 👍 will update